contributing/Makefile

```diff
@@ -0,0 +1,34 @@
+synthetic-data/*.rif:
+	./fetch.sh
+
+.env:
+	cp .env.example .env
```

Comment:
This doesn't work since .env.example doesn't actually exist:
```
make .env
cp .env.example .env
cp: .env.example: No such file or directory
make: *** [.env] Error 1
```

Reply:
removed with added instructions in readme.